backend/tests/v5/unit/services/mailer/templates/baseTemplate.test.js

```diff
@@ -0,0 +1,960 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const { src } = require('../../../../helper/path');
+const { generateRandomString } = require('../../../../helper/services');
+
+const BaseTemplate = require(`${src}/services/mailer/templates/baseTemplate`);
+
+const testHtml = () => {
+	describe('get base template html', () => {
+		test('should get base template html', async () => {
+			const data = {
+				firstName: generateRandomString(),
+				emailContent: generateRandomString(),
+			};
+
+			const res = await BaseTemplate.html(data);
+
+			expect(res).toEqual(
```

Comment:
This is a bit excessive

could we just check if the string is an html? https://www.npmjs.com/package/is-html

Also we should check data was provided has all the data we need, so we don't send out dodgy emails